public/app/features/explore/state/helpers.ts

```diff
@@ -1,4 +1,5 @@
-import { DefaultTimeZone, TimeRange, toUtc, SupplementaryQueryType } from '@grafana/data';
+import { TimeRange, toUtc, SupplementaryQueryType } from '@grafana/data';
+import { defaultTimeZone } from '@grafana/schema';
 
```

Comment:
what's up with all of these import changes? 🤔

Reply:
there are failing tests, so i'm trying to backport a possibly related cleanup to try to fix.